Mobl_ios_Callisto-master/MealTrackingPilot/Authentication/LoginFormViewController.swift

```diff
@@ -146,7 +145,7 @@ class LoginFormViewController: UIViewController, UITextFieldDelegate {
         viewModel.status.subscribe(onNext: { [weak self] (_) in
             guard let welf = self else { return }
```

Comment:
Remove this line guard let welf = self else { return }

Reply:
done changes